keyboards/treadstone48/rev1/rev1.h

```diff
@@ -0,0 +1,187 @@
+#ifndef REV1_H
+#define REV1_CONFIG_H
+
+#include "../treadstone48.h"
+
+//void promicro_bootloader_jmp(bool program);
+#include "quantum.h"
+
+#ifdef RGBLIGHT_ENABLE
+//rgb led driver
+#include "ws2812.h"
+#endif
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+  #include <avr/io.h>
+  #include <avr/interrupt.h>
+#endif
+#endif
+
+// full Keymap
+// (TRRS jack on the left half is to the right, TRRS jack on the right half is to the left)
+#define LAYOUT_FULL( \
```

Comment:
The QMK convention is to label the LAYOUTS with `LAYOUT_some_lower_case_description`. Could you change all these to lower case?

```suggestion
#define LAYOUT_full( \
```